config/manifests/bases/rhpam-kogito-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
We are missing a lot of information in this file.

Reply:
updated it now. please recheck